openwebstart/src/main/java/com/openwebstart/jvm/ui/RuntimeManagerPanel.java

```diff
@@ -168,7 +167,7 @@ private boolean handleSuccessfulFoundRuntime(final Path path, final LocalJavaRun
 
     private void onFindLocalRuntimes() {
         LOG.info("Starting to search for local JVMs");
-        backgroundExecutor.execute(() -> {
+        getDaemonExecutorService().execute(() -> {
```

Comment:
It won't be a problem to have this one as a non daemon executor. I prefer to change as less behaviour as possible with this PR so we should change it here to non daemon.